Knysh/Dijkstra/dj.C

```diff
@@ -0,0 +1,58 @@
+// spiszhenno, no zato razobralsa
+#include <iostream>
+#include <vector>
+#define INT_MAX 100000
+using namespace std;
+
+void Dijkstra(const std::vector<std::vector<int> >& GR, int st)
```

Comment:
naming convention for functions is `camelCase`.
What is `GR` and `st`? Give meaningful names for function arguments.